src/java/picard/sam/markduplicates/MarkDuplicates.java

```diff
@@ -64,28 +64,35 @@
-                    "artifacts or from bio-chemical processes during clonal amplification and sequencing; they are identified using the\n" +
-                    "READ_NAME_REGEX and the OPTICAL_DUPLICATE_PIXEL_DISTANCE options.\n" +
-                    "\n" +
-                    "The tool's main output is a new SAM or BAM file in which duplicates have been identified in the SAM flags field, or\n" +
-                    "optionally removed (see REMOVE_DUPLICATE and REMOVE_SEQUENCING_DUPLICATES), and optionally marked with a duplicate type\n" +
-                    "in the 'DT' optional attribute. In addition, it also outputs a metrics file containing the numbers of\n" +
-                    "READ_PAIRS_EXAMINED, UNMAPPED_READS, UNPAIRED_READS, UNPAIRED_READ DUPLICATES, READ_PAIR_DUPLICATES, and\n" +
-                    "READ_PAIR_OPTICAL_DUPLICATES.\n" +
-                    "\n" +
-                    "Usage example: java -jar picard.jar MarkDuplicates I=input.bam \\\n" +
-                    "                 O=marked_duplicates.bam M=marked_dup_metrics.txt\n" +
-                    "\n" +
-                    "The program can take either coordinate-sorted or query-sorted input, however the behavior is slightly different.\n" +
-                    "When the input is coordinate-sorted, unmapped mates of mapped records, and supplementary and secondary alignments are not\n" +
-                    "marked as duplicates. When the input is query-sorted (actually query-grouped) then unmapped mates get marked as their mapped\n" +
-                    "counter-parts, and secondary and supplementary reads get marked as the primary records with the same query-name.\n";
+    static final String USAGE_DETAILS = "<p>This tool locates and tags duplicate reads in a BAM or SAM file, where duplicate reads are " +
+            "defined as originating from a single fragment of DNA.  Duplicates can arise during sample preparation " +
+            "e.g. library construction using PCR.  For additional information on PCR-associated duplication artifacts, please see the documentation for the <a href='https://broadinstitute.github.io/picard/command-line-overview.html#EstimateLibraryComplexity'>EstimateLibraryComplexity</a> tool.  " +
+            "Alternatively, duplication artifacts can also arise from reads derived from a single amplification cluster, incorrectly detected as multiple clusters by the optical sensor of the sequencing instrument.  These duplication artifacts are referred to as optical duplicates.</p>" +
+            "" +
+            "<p>The READ_NAME_REGEX and the OPTICAL_DUPLICATE_PIXEL_DISTANCE arguments are the primary methods to identify and differentiate duplicate types.  " +
+            "The tool's main output is a new SAM or BAM file, in which duplicates have been identified in the SAM flags field.  Duplicates are marked using the 'DT' tag in the 'optional field' section of a SAM/BAM file.</p>"+
```

Comment:
If you want to talk about DT that's fine but you have to disambiguate what it's for.